va/va.go

```diff
@@ -200,9 +239,11 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 
 	dialer, prob := va.resolveAndConstructDialer(ctx, host, port)
 	dialer.record.URL = url.String()
-	validationRecords := []core.ValidationRecord{dialer.record}
+	// Start with an empty validation record list - we will add a record after
+	// each dialer.Dial()
+	validationRecords := []core.ValidationRecord{}
```

Comment:
You can't go from `nil` -> slice, if you want to start with a `nil` slice you'd want `var validationRecords []core.ValidationRecord{}`.